packages/engine-rn-tvos/package.json

```diff
@@ -44,7 +44,8 @@
         "@babel/preset-env": "7.8.7"
     },
     "peerDependencies": {
-        "rnv": "^0.37.0-canary.11"
+        "rnv": "^0.37.0-canary.11",
+        "metro-cache": "^0.66.2"
```

Comment:
@mihaiblaga89 can you add this to every engine which requires metro-cache to avoid disrepancies